app/components/ui/molecules/DashboardListItem.md

```diff
@@ -1,11 +1,44 @@
-List item in the dashboard that links to manuscript. It shows status (e.g. accepted, waiting approval), manuscript title and date since status last changed.
```

Comment:
FYI, I've changed the description slightly because this component doesn't know anything about linking to a manuscript - this logic is handled in `DashboardList`